apps/sms/style/report_view.css

```diff
@@ -113,16 +110,16 @@
   padding-top: 0.9rem;
 }
 
-.report-view > .contact-list div {
+.report-view > .contact-list .network-status {
```

Comment:
nit: Maybe we don't need the `.contact-list` in selector since you already set a specific class for status block?